go-sdk/instrumentation/otelopenai/config.go

```
@@ -0,0 +1,63 @@
+package otelopenai
+
+import (
+	"go.opentelemetry.io/otel/propagation"
+	oteltrace "go.opentelemetry.io/otel/trace"
+)
+
+// config is used to configure the middleware.
+type config struct {
+	tracerProvider                oteltrace.TracerProvider
+	propagators                   propagation.TextMapPropagator
+	traceIDResponseHeaderKey      string
```

golangci-lint (1.64.8)

field `traceIDResponseHeaderKey` is unused

(unused)